app/experimenter/nimbus-ui/src/lib/experiment.ts

```diff
@@ -10,7 +10,7 @@ export function getStatus(
   experiment?: getExperiment_experimentBySlug | getAllExperiments_experiments,
 ) {
   const status = experiment?.status;
-
+  const ending = !!experiment?.isEndRequested;
```

Comment:
I see the `status.live` check in the `Summary` component, though this should probably be `status === NimbusExperimentStatus.LIVE && !!experiment?.isEndRequested` since that's more exactly what the state will be. But, nothing functional here would change with that, just might clarify things vaguely in the future.

Reply:
Fair point